src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,718 @@
+
+    ```astro title="src/components/Head.astro" ins={2, 5}
+    ---
+    import { Font } from 'astro:assets'
+    ---
+
+    <Font cssVariable="--font-roboto" preload />
+    ```
+
+    This is commonly done in a component such as `Head.astro` that is used in a common site layout.
+
+    <ReadMore>See the full [`<Font>` component reference](#font--component-reference) for more information.</ReadMore>
+
+    Since the `<Font />` component generates CSS with font declarations, you can reference the font family using the `cssVariable`:
+
+    <Tabs>
+
+    <TabItem label="CSS">
+
+    ```css ins={3}
+    <style>
+    body {
+        font-family: var(--font-roboto);
+    }
+    </style>
+    ```
+
+    </TabItem>
+
+    <TabItem label="Tailwind CSS 4.0">
+
+    ```css title="src/styles/global.css" ins={4}
+    @import 'tailwindcss';
+
+    @theme {
+        --font-sans: var(--font-roboto);
+    }
+    ```
+
+    </TabItem>
+
+    <TabItem label="Tailwind CSS 3.0">
+
+    ```js title="tailwind.config.mjs" ins={6-8}
+    /** @type {import("tailwindcss").Config} */
+    export default {
+    content: ["./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}"],
+    theme: {
+        extend: {},
+        fontFamily: {
+            sans: ["var(--font-roboto)"]
+        }
+    },
+    plugins: [],
+    };
+    ```
+
+    </TabItem>
+
+    </Tabs>
+
+</Steps>
+
+## Available remote font providers
+
+Astro re-exports most [unifont](https://github.com/unjs/unifont/) providers. The following have built-in support:
+
+- [Adobe](https://fonts.adobe.com/)
+- [Bunny](https://fonts.bunny.net/)
+- [Fontshare](https://www.fontshare.com/)
+- [Fontsource](https://fontsource.org/)
+- [Google](https://fonts.google.com/)
+
+To use a built-in remote provider, configure `provider` with the appropriate value for your chosen font provider:
+
+<Tabs>
+
+<TabItem label="Adobe">
+
+```js
+provider: fontProviders.adobe({id: process.env.ADOBE_ID})
+```
+
+</TabItem>
+
+<TabItem label="Bunny">
+
+```js
+provider: fontProviders.bunny()
+```
+
+</TabItem>
+
+<TabItem label="Fontshare">
+
+```js
+provider: fontProviders.fontshare()
+```
+
+</TabItem>
+
+<TabItem label="Fontsource">
+
+```js
+provider: fontProviders.fontsource()
+```
+
+</TabItem>
+
+<TabItem label="Google">
+
+```js
+provider: fontProviders.google()
+```
+
+</TabItem>
+
+</Tabs>
+
+You can also [make a custom Astro font provider](#build-your-own-font-provider) for any unifont provider.
+
+## `<Font />` component reference
+
+This component outputs style tags and can optionally output preload links for a given font family.
+
+It must be imported and added to your page `<head>`. This is commonly done in a component such as `Head.astro` that is used in a common site layout for global use, but may be added to individual pages as needed.
+
+With this component, you have control over which font family is used on which page, and which fonts are preloaded.
+
+### cssVariable
+
+<p>
+
+**Type:** `string`
```

Comment:
```suggestion
**Example type:** `"--font-roboto" | "--font-open-sans" | ...`
```
I didn't know whether using comic sans as an example was too controversial 😅 I just wanted something with two words. Could also choose `--font-times-new-roman`

Reply:
Honestly comic sans would be pretty funny

Reply:
It's your feature, I leave it to you to make the final call! 😂